core/shared/utils/runtime_timer.h

```diff
@@ -15,7 +15,7 @@ extern "C" {
 uint64
 bh_get_tick_ms(void);
 uint32
-bh_get_elpased_ms(uint32 *last_system_clock);
+bh_get_elapsed_ms(uint32 *last_system_clock);
```

Comment:
this is an api breaking change.
wamr-app-framework seems using this. can you take care of it?

Reply:
will do

Reply:
what's the plan for addressing it? Are we leaving it as is, or we create a new API and mark the existing one as deprecated/to be removed in the next major release?